Skip test that uses IBus, since it tries to access /var/lib/dbus/machine-id which doesn't exist on the system.

diff --git a/tests/meson.build b/tests/meson.build
index b814d59..bf30b96 100644
--- a/tests/meson.build
+++ b/tests/meson.build
@@ -28,13 +28,10 @@ test_names = [
   'test-expander-row',
   'test-header-bar',
   'test-header-group',
-  'test-keypad',
   'test-paginator',
   'test-preferences-group',
   'test-preferences-page',
   'test-preferences-row',
-  'test-preferences-window',
-  'test-search-bar',
   'test-squeezer',
   'test-string-utf8',
   'test-swipe-group',
